lib/migrate.js

```diff
@@ -142,7 +129,31 @@ function mixinMigrate(CouchDB) {
   };
 
   /**
-   * Add an index for modelView propery to indexes got from modelDef
+    * Used in function `migrateOrUpdateIndex`.
+    * Create index with index fields.
+    * @param {Object} mo  The model configuration.
+    * @param {String} model  The model name.
+    * @param {String} name  The index name.
+    * @param {Object} indexObj The index object e.g. [{"foo": "asc"}, {"bar": "asc"}]
+    * @param {Function} cb
+  */
+  CouchDB.prototype.__createIndex = function(mo, model, name, fields, cb) {
```

Comment:
I think we usually use a single underscore for private members, e.g. `_createIndex`. Is there any particular reason for using two underscores?

Reply:
good catch 🙈